### ISSUE TITLE
Provide a way to query for daylight savings time

### ISSUE THREAD

**Issue by rilerez** [juxt/tick#116]
This seems like something that's hard to get right ad-hoc, but exists in the timezone databases.

**Comment by malcolmsparks** [comment]
Could you provide a little more detail. For example, are you looking for a way to get the date of DST changes?

**Comment by rilerez** [comment]
Yes that was it! I found a way to do it by searching through a range of zoned-date-times, but is there a better way in cljc.java-time?

**Comment by henryw374** [comment]
Maybe zone rules?
https://stackoverflow.com/questions/38736793/determine-future-timezone-transitions

**Comment by rilerez** [comment]
Thank you!
